drivers/scheduler/k8s/specs/nginx/px-nginx-storage.yaml

```diff
@@ -6,14 +15,16 @@ metadata:
 provisioner: kubernetes.io/portworx-volume
 parameters:
    repl: "3"
-   shared: "true"
```

Comment:
we can't do this. This reduces shared volumes coverage.

Reply:
Use other existing apps like mysql, postgres or elasticsearch which already don't have shared:true

Reply:
Isn't wordpress currently using shared?

Reply:
Yes, but we need multiple shared apps. We already have postgres, mysql, elasticsearch and wordpress-mysql for regular volumes. So need equal coverage for shared volumes.  

The current torpedo`wordpress` stack is a complex beast and we have to disable that many times due to app issues. Also it has shared and non-shared volumes in the same stack so we need nginx for only shared volume tests.

Reply:
It does not make sense to have nginx only on shared just because wordpress is using both shared and non-shared together. From a testing shared volumes perspective, I don't see any difference in a stack using both volume type vs a stack that using only one.
I am assuming we are running multiple apps together anyway.

If wordpress is disabled, then yes we loose coverage for shared volumes.
I will create another spec directory nginx-enc for this use case.